tests/framework/PostProcessors/DataMiningPostProcessor/DimensionalityReduction/test_dataMiningLocallyLinearEmbedding.xml

```diff
@@ -31,8 +31,8 @@
       <KDD labelFeature="component" lib="SciKitLearn">
         <Features>x1,x2,x3,x4</Features>
         <SKLtype>manifold|LocallyLinearEmbedding</SKLtype>
-        <eigen_solver>dense</eigen_solver>
-        <tol>1E-10</tol>
+        <eigen_solver>arpack</eigen_solver>
```

Comment:
is it listed in user manual?

Reply:
Yes
![Screen Shot 2023-02-16 at 3 22 55 PM](https://user-images.githubusercontent.com/9202671/219500641-bb11b8fc-3fea-4ff0-b5a9-643f02146f35.png)